models/pathology_nuclick_annotation/configs/metadata.json

```diff
@@ -0,0 +1,79 @@
+{
+    "schema": "https://github.com/Project-MONAI/MONAI-extra-test-data/releases/download/0.8.1/meta_schema_20220324.json",
+    "version": "0.0.1",
+    "changelog": {
+        "0.0.1": "initialize the model package structure"
+    },
+    "monai_version": "1.0.1",
+    "pytorch_version": "1.13.0",
+    "numpy_version": "1.21.2",
+    "optional_packages_version": {
+        "nibabel": "4.0.1",
+        "pytorch-ignite": "0.4.9"
+    },
+    "task": "Pathology Nuclick segmentation",
+    "description": "A pre-trained model for Nuclei Classification within Haematoxylin & Eosin stained histology images",
+    "authors": "MONAI team",
+    "copyright": "Copyright (c) MONAI Consortium",
+    "data_source": "consep_dataset.zip from https://warwick.ac.uk/fac/cross_fac/tia/data/hovernet",
+    "data_type": "png",
+    "image_classes": "RGB channel data, intensity scaled to [0, 1]",
+    "label_classes": "single channel data",
+    "pred_classes": "2 channels OneHot data, channel 1 is nuclei, channel 0 is background",
+    "eval_metrics": {
+        "mean_dice": 0.96
```

Comment:
Similar question, I did not find the same number in readme, thus does the number wrong here?

Reply:
Fixed